src/lib/sidebar-components/Feedback/FeedbackCategoryButton.tsx

```diff
@@ -0,0 +1,37 @@
+import clsx from "clsx"
+import React from "react"
+import { FeedbackCategory, FeedbackContent } from "./Feedback"
+
+type Props = {
+  feedbackCategory: FeedbackCategory
+  feedbackContent: FeedbackContent
+  setFeedbackContent: (value: React.SetStateAction<FeedbackContent>) => void
+}
+
+export default function FeedbackCategoryButton({
+  feedbackCategory,
+  feedbackContent,
+  setFeedbackContent,
+}: Props) {
+  const active = feedbackCategory === feedbackContent.feedbackCategory
```

Comment:
@joobacca Muss active hier nicht ein State sein, damit es reactive ist und die UI auf die Changes reagiert? Oder ist das egal, weil der Value über Props kalkuliert wird und die Props sind ja reactive.

Reply:
Ja genau, die Variable active wird in diesem Fall bei jedem Render mit den aktuellen Werten neu berechnet. Einen State bräuchtest du in diesem Fall nur, wenn du bei Änderungen bei der active-Variable auch rerendern wollen würdest